M2/Macaulay2/packages/Macaulay2Doc/doc12.m2

```diff
@@ -629,37 +629,34 @@ document {
      Key => {ScriptedFunctor,
 	  (symbol^, ScriptedFunctor, Thing),
 	  (symbol_, ScriptedFunctor, Thing),
-	  (symbol SPACE, ScriptedFunctor, Thing)},
+	  Functor, (symbol SPACE, Functor, Thing)},
      Headline => "the class of all scripted functors",
      "A scripted functor accepts a subscript or a superscript:
      the primary example is ", TO "HH", ".",
-     SeeAlso => {"subscript", "superscript", "argument"}
+     Subnodes => TO \ {"subscript", "superscript", "argument"}
      }
 
 document {
      Key => argument,
-     Headline => "specify the function in a scripted functor for an argument",
-     TT "argument", " -- a key used in scripted functors under which is
+     Headline => "specify the function in a functor for an argument",
+     TT "argument", " -- a key used in functors under which is
      stored the function that accepts the arguments.",
-     SeeAlso => "ScriptedFunctor"
```

Comment:
If you remove this line, then the reader of this documentation page has no quick way to get to the documentation for scripted functors.

Reply:
These pages are subnodes, so there is a link displayed at the top.

Reply:
Here you go:
https://github.com/Macaulay2/M2/blob/0596fa84d9d169f5d6ec2a57072c25933a0941ab/M2/Macaulay2/packages/Macaulay2Doc/doc12.m2#L635
https://github.com/Macaulay2/M2/blob/0596fa84d9d169f5d6ec2a57072c25933a0941ab/M2/Macaulay2/packages/Macaulay2Doc/doc12.m2#L645
Did you try rebuilding the documentation to see this?
![image](https://user-images.githubusercontent.com/3147718/156045837-b6bf7a71-d02d-48e2-bd9e-b62d8e9faa42.png)

Reply:
That's only visible in the html and info versions of the documentation.  If you type `help superscript`, you won't see it.

Reply:
That sounds like an issue that should be fixed in `help`, since `viewInfo` also shows the right thing. I made an issue for it.